scipy/sparse/linalg/_eigen/arpack/arpack.py

```diff
@@ -321,7 +321,11 @@ def __init__(self, n, k, tp, mode=1, sigma=None,
             raise ValueError("maxiter must be positive, maxiter=%d" % maxiter)
 
         if tp not in 'fdFD':
-            raise ValueError("matrix type must be 'f', 'd', 'F', or 'D'")
+            # Use `float64` libraties from integer dtypes.
```

Comment:
Typo "libraties"? Couldn't quite work out what this was meant be

Reply:
Yes, a typo, "libraries".  This module is not a solver per se but an API to one of the precompiled libraires chosen depending on the types of the input parameters. Here, for the integer type of the input matrix, earlier unsupported, I call the same library as for the 'float64' matrix, as requested in https://github.com/scipy/scipy/pull/19209#discussion_r1320852204 , to my recollection

Reply:
```suggestion
            # Use `float64` libraries from integer dtypes.
```